libs/exec/execv.go

```diff
@@ -10,8 +12,21 @@ type ExecvOptions struct {
 
 	// Dir is the working directory of the child process.
 	Dir string
+
+	// It is not possible to execute a cmd.exe script inlined as a argument
+	// to cmd.exe. They have to be serialized to a file and executed.
+	// Thus if [Execv] is used to execution a script though cmd.exe,
+	// the caller should register a cleanup function to clean up the temporary file.
+	windowsCleanup func()
+
+	// Callback to exit the current process in windows. Having this as a function here
+	// helps with testing.
+	windowsExit func(status int)
```

Comment:
Since it's only for the tests, is there an option to have an acceptance test instead which exercise this path?

Reply:
For the status code, yes, but not for the cleaning up of the scripts part, which is the more important aspect here,

We already have acceptance tests for the exit code.

Reply:
But why can't we check that cleaned up happened? We set the TMPDIR anyway, so we know which folder we need to check and the cleanup is synchronous. Any reasons which does not allow it?

Reply:
Another problem is we do not use the ShellExec function anywhere yet. We'll use it once we add the scripts section to DABs. 

It's nice to have dediced unit tests for this, rather than eventually test this once we have script execution via a `scripts` section.